docs/introduction/QuickStart.md

```diff
@@ -0,0 +1,200 @@
+
+Well, that's where `createSlice` from Redux Toolkit comes in :)
+
+```js
+// before
+const add = createAction('ADD')
+const substract = createAction('SUBSTRACT')
+
+const calculator = createReducer(0, {
+  [add]: (state, action) => state + action.payload,
+  [subtract]: (state, action) => state - action.payload
+})
+
+// after
+const calculatorSlice = createSlice({
+  name: 'calculator',
+  initialState: 0,
+  reducers: {
+    add: (state, action) => state + action.payload,
+    subtract: (state, action) => state - action.payload
+  }
+})
+```
+
+Now the `add` and `subtract` actions will be available via `calculatorSlice.actions` and the reducer function will be available via `calculatorSlice.reducer`. This keeps our "slice" of the store all in one place, so that we'll know where all of the logic for this part of the store resides.
+
+Awesome, let's move onto setting up the store.
+
+### The Store
+
+```js
+const store = configureStore({
+  reducer: calculatorSlice.reducer
+})
+```
+
+Using Redux Toolkit's `configureStore`, we can setup the store with helpful default middleware out of the box. Unfortunately, this example won't be able to highlight this middleware, but be sure to check out the [included default middleware](https://redux-starter-kit.js.org/api/getDefaultMiddleware#included-default-middleware) to learn more about them.
+
+To Recap, here's the full code that we've written so far:
+
+```js
+const calculatorSlice = createSlice({
+  name: 'calculator',
+  initialState: 0,
+  reducers: {
+    add: (state, action) => state + action.payload,
+    subtract: (state, action) => state - action.payload
+  }
+})
+
+const store = configureStore({
+  reducer: calculatorSlice.reducer
+})
+```
+
+Not too bad :)
+
+## Vanilla Example
+
+Great! Now that we have the redux portion of the code done, let's move onto creating an application that can utilize it.
+
+```js
+const valueEl = document.getElementById('value')
+
+const render = function() {
+  valueEl.innerHTML = store.getState().toString()
+}
+
+// we'll need to render it once to populate the value element with some data
+render()
+
+// Setup the subscription to the value element
+store.subscribe(render)
+
+// let's destructure the actions from the calculator slice
+const { add, subtract } = calculatorSlice.actions
+
+// Setup event listeners for the buttons
+document.getElementById('add').addEventListener('click', function() {
+  const value = document.getElementById('input').value
+  store.dispatch(add(value))
+})
+
+document.getElementById('subtract').addEventListener('click', function() {
+  const value = document.getElementById('input').value
+  store.dispatch(subtract(value))
+})
+```
+
+alright, now we have a simple app setup that utilizes our Redux code to have a functioning calculator app. it will now dispatch actions to update the value in the store,
+depending on which button was clicked.
```

Comment:
This makes feel like I'm wanting more out of this explanation. Adding external resources might help? Or just a more in-depth explanation.

Reply:
Also needs a runnable CodeSandbox here.